test/integration/http2_integration_test.cc

```diff
@@ -1772,10 +1773,14 @@ TEST_P(Http2FloodMitigationTest, ZerolenHeader) {
   EXPECT_EQ(1, test_server_->counter("http2.rx_messaging_error")->value());
   EXPECT_EQ(1,
             test_server_->counter("http.config_test.downstream_cx_delayed_close_timeout")->value());
+  EXPECT_THAT(waitForAccessLog(access_log_name_), HasSubstr("http2.invalid.header.field"));
+  // expect Downstream Protocol Error
```

Comment:
nit: expect Downstream Protocol Error -> Expect a downstream protocol error.

Reply:
ack, will be in the next push